cmp/testdata/diffs

```diff
@@ -155,8 +155,8 @@
 >>> TestDiff/Comparer#42
 <<< TestDiff/Comparer#43
   map[*int]string{
-- 	⟪0xdeadf00f⟫: "hello",
-+ 	⟪0xdeadf00f⟫: "world",
+- 	&⟪0xdeadf00f⟫0: "hello",
```

Comment:
What/where is the ending 0 coming from in &⟪0xdeadf00f⟫0?

Reply:
It's the value of the pointed-at value. Previously, it was elided, which is confusing.